File: python/edition.py
class Edition:
    def __init__(self, title, url, price, discount, sales_type, data_id):
        self.title = title
        self.url = url
        self.price = price
        self.discount = discount
        self.sales_type = sales_type
        self.data_id = data_id

    def __str__(self):
        return "Title: " + self.title + "\n" + "URL: " + self.url + "\n" + "Price: " + self.price + "\n" + "Discount: " + self.discount + "Sales Type: " + self.sales_type + "\n" + "Data-ID: " + self.data_id

File: SalesTracker/SalesTracker/Models/Edition.cs
using System.ComponentModel.DataAnnotations;

namespace SalesTracker.Models
{
    public class Edition
    {
        [Key]
        public int Id { get; set; }

        [MaxLength(250)]
        public string Title { get; set; } = String.Empty;
        [MaxLength(1000)]
        public string URL { get; set; } = String.Empty;
        [MaxLength(20)]
        public string Price { get; set; } = String.Empty;
        [MaxLength(20)]
        public string Discount { get; set; } = String.Empty;

        public DateTime LastUpdated { get; set; }

        public virtual SaleType SaleType { get; set; } = null!;

        public bool IsDeleted { get; set; }

        public string DataId { get; set; } = String.Empty;
    }
}
